addons/ofxOsc/src/ofxOscMessage.cpp

```diff
@@ -142,6 +154,26 @@ float ofxOscMessage::getArgAsFloat( int index ) const
 }
 
 
+double ofxOscMessage::getArgAsDouble( int index ) const
+{
+    if ( getArgType(index) != OFXOSC_TYPE_DOUBLE )
+    {
+        if ( getArgType( index ) == OFXOSC_TYPE_INT32 )
+        {
+            ofLogWarning("ofxOscMessage") << "getArgAsDouble(): converting double to int32 for argument " << index;
+            return ((ofxOscArgInt32*)args[index])->get();
+        }
+        else
+        {
+            ofLogError("ofxOscMessage") << "getArgAsDouble(): argument " << index << " is not a number";
```

Comment:
what about the other number types (FLOAT, INT64) - those are not accounted for, and would erroneously trip this error.